tests/login.test.js

```diff
@@ -0,0 +1,146 @@
+
+    it("POST /Login  Bad Request", (done) =>{
+        const login = {
+            username: "    ",
+            password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("POST /Login  Bad Request Password", (done) =>{
+        const login = {
+            username: "Karla",
+            password: "Mmol157200#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La contraseña es inválida");
+            done();
+        });
+    });
+
+    it("POST /Login Forbidden", (done) =>{
+        const login = {
+            username: "Miroslava25",
+            password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            response.should.have.status(403);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La cuenta esta bloqueada o esta inactiva comuniquese con el administrador");
+            done();
+        });
+    });
+
+    it("POST /Login Not Found", (done) =>{
+        const login = {
+            username: "Karls",
+            password: "Marst1245#"
+        };
+        chai.request(server).post("/login").send(login)
+        .end( (error, response) => {
+            response.should.have.status(404);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("No se encontro la cuenta");
+            done();
+        });
+    });
+});
+
+describe("PATCH Login",() =>{
+    it("PATCH /Login", (done) =>{
+        const login = {
+            username: "MiroStar",
+            codeConfirmation: 234990
+        };
+        chai.request(server).patch("/login").send(login)
+        .end( (error, response) => {
+            response.should.have.status(201);
+            response.body.should.have.property("message");
+            response.body.should.have.property("message").eq("La confirmación es exitosa");
+            done();
+        });
+    });
+
+    it("POST /Login  Bad Request", (done) =>{
+        const login = {
+            username: "   ",
+            codeConfirmation: 123
+        };
+        chai.request(server).patch("/login").send(login)
+        .end( (error, response) => {
+            response.should.have.status(400);
+            response.body.should.be.a("object");
+            response.body.should.have.property("errors");
+            done();
+        });
+    });
+
+    it("POST /Login  Bad Request Code", (done) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space after =>.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    it("POST /Login  Bad Request Code", (done) => {
```